code/modules/mob/living/simple_animal/hostile/mining/elites/elite.dm

```diff
@@ -339,6 +345,29 @@ While using this makes the system rely on OnFire, it still gives options for tim
 		visible_message("<span class='warning'>[mychild] suddenly reappears above [src]!</span>")
 		playsound(loc,'sound/effects/phasein.ogg', 200, 0, 50, TRUE, TRUE)
 
+/obj/structure/elite_tumor/proc/invaders_check()
+	for(var/mob/living/M in range(12, src))
```

Comment:
Could potentially be an expensive thing to keep running. Would maybe be worth it to check if `/datum/component/proximity_monitor` is better performing or not. Or maybe by making a new area for this arena. On which you listen to `area/Entered(A)`

Reply:
Proximity used!